### PR TITLE
TT-120: Modify cursor in Charges/Minutes selector so people know they're buttons. (FE)

### PR DESCRIPTION
**Pull request by elijah-parrish**
Changed lines 24 and 25 in committee.vue to change the cursor to a pointing hand when the user mouses over the charges and meetings buttons.

### CODE REVIEW

File: src/pages/Minutes.vue [hunk 1]
```diff
@@ -3,10 +3,18 @@
     <link href="https://cdn.quilljs.com/1.3.6/quill.snow.css" rel="stylesheet">
     <HeaderMenu />
     <CommitteesMenu />
-        <div class="pagename" :style="{ 'background-image': 'url(' + backgroundImage + ')' }">
+    <div class="pagename" :style="{ 'background-image': 'url(' + backgroundImage + ')' }">
```
Comment (omardlhz) [inline src/pages/Minutes.vue]: I dont know if this changes should be here?

Comment (omardlhz) [reply src/pages/Minutes.vue]: Was this file meant to be edited?

**Comment by omardlhz** [comment]
Hey @elijah-parrish, it appears that there are merge conflicts. Could you please run `git rebase origin/master` on this branch? Thank you!

File: src/pages/Minutes.vue (hunk 1 above)
Comment (elijah-parrish) [reply src/pages/Minutes.vue]: This was resolved today

File: src/pages/Minutes.vue [hunk 2]
```diff
@@ -19,7 +19,6 @@
 import HeaderMenu from '../components/HeaderMenu'
 import CommitteesMenu from '../components/CommitteesMenu'
 import MinutesControls from '../components/MinutesControls'
-import Auth from '../mixins/auth'
```
Comment (omardlhz) [inline src/pages/Minutes.vue]: I think you might've accidentally removed this line. This is needed for authentication.

File: src/pages/Minutes.vue [hunk 3]
```diff
@@ -184,4 +184,4 @@ export default {
     from { opacity: 0; }
     to   { opacity: 1; }
   }
-</style>
+</style>
```
Comment (omardlhz) [inline src/pages/Minutes.vue]: Please add new line in end of file.

File: src/pages/Minutes.vue (hunk 2 above)
Comment (elijah-parrish) [reply src/pages/Minutes.vue]: This was a merge conflict, all good now

Comment (omardlhz) [reply src/pages/Minutes.vue]: This line still appears to not be there, please make sure you don't remove it!

**Pull request merged**